src/parser.rs

```diff
@@ -88,8 +86,9 @@ fn search_for_attr(
     exact_depth: Option<usize>,
 ) -> Result<Vec<(NixNode, String, usize)>, String> {
```

Comment:
We should pull in https://github.com/yaahc/color-eyre and https://lib.rs/crates/anyhow for (user facing) errors

Reply:
This probably belongs as a separate feature draft. Do you envision color-eyre dumping errors to stderr or being passed in through skim?

Reply:
As `color-eyre` is an panic handler, I don't think it should be passed in through skim.

Reply:
Should be passed to skim since the user will need to see it.

Reply:
We may want to draw some sort of distinction between unrecoverable and recoverable errors and dump accordingly. Maybe unrecoverable -> stderr, recoverable -> skim? I don't know how tricky this will be to pass to skim. In any case, I'll move to a separate issue.